keps/sig-node/4639-oci-volume-source/README.md

```diff
@@ -1008,12 +1040,15 @@ rollout. Similarly, consider large clusters and how enablement/disablement
 will rollout across nodes.
 -->
 
+kube-apiserver must support it, and every node that has a pod scheduled that attempts to mount image volumes must have a kubelet and CRI that support it.
```

Comment:
as a practical matter, how can this be enforced.

Reply:
I can only imagine it could be enforced in the mixed support situation (apiserver has support but kubelets don't), in which case an admission webhook or scheduling plugin could be added. If the field is getting filtered before the pod is processed, I'm not sure there's anything an admin could do. do I understand the request correctly?

Reply:
This mixed support situation will happen for an indeterminate period of time on every upgrading cluster. Describing specifically what happens during this period is important.

What is bad behavior they should anticipate?

Reply:
I think this is highly related to the `Version Skew Strategy` above, I added more context to the section to clarify how the kubelet behaves if the feature is not available there.